UserExperience_Remove_CssJs_Page_View.js

```diff
@@ -27,21 +27,32 @@ jQuery(function() {
 				);
 
 				if (!exists) {
+					var single_id = single_path.replace(/[^\w-]/g, '_');
```

Comment:
>Variable and function names should be full words, using camel case with a lowercase first letter.

https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/#naming-conventions

Reply:
Updated all variable and methods to use camel case